test/integration/db/db_tests.dart

```diff
@@ -458,183 +458,186 @@ runTests(db.DatastoreDB store, String namespace) {
+              },
+              // Expando queries: Filter on expanded int property
+                  () async {
+                var query = store.query<ExpandoPerson>(partition: partition)
                   ..filter('bar =', (expandoPersons.last as dynamic).bar)
                   ..run();
-              var models = await runQueryWithExponentialBackoff(query, 1);
-              compareModels([expandoPersons.last], models);
-            },
-            // Expando queries: Filter normal property with different
-            // propertyName (datastore name is 'NN').
-            () async {
-              var query = store.query<ExpandoPerson>(partition: partition)
+                var models = await runQueryWithExponentialBackoff(query, 1);
+                compareModels([expandoPersons.last], models);
+              },
+              // Expando queries: Filter normal property with different
+              // propertyName (datastore name is 'NN').
+                  () async {
+                var query = store.query<ExpandoPerson>(partition: partition)
                   ..filter('nickname =', expandoPersons.last.nickname)
                   ..run();
-              var models = await runQueryWithExponentialBackoff(query, 1);
-              compareModels([expandoPersons.last], models);
-            },
-
-            // Delete results
-            () => store.commit(deletes: allKeys),
-
-            // Wait until the entity deletes are reflected in the indices.
-            () => waitUntilEntitiesGone(store, allKeys, partition),
-
-            // Make sure queries don't return results
-            () => store.lookup(allKeys).then((List<db.Model> models) {
-              expect(models.length, equals(allKeys.length));
-              for (var model in models) {
-                expect(model, isNull);
-              }
-            }),
-          ];
-          return Future.forEach(tests, (f) => f());
+                var models = await runQueryWithExponentialBackoff(query, 1);
+                compareModels([expandoPersons.last], models);
+              },
+
+              // Delete results
+                  () => store.commit(deletes: allKeys),
+
+              // Wait until the entity deletes are reflected in the indices.
+                  () => waitUntilEntitiesGone<User>(store, userKeys, partition),
+                  () => waitUntilEntitiesGone<ExpandoPerson>(store, expandoPersonsKeys, partition),
+
+              // Make sure queries don't return results
+                  () => store.lookup(allKeys).then((List<db.Model> models) {
+                expect(models.length, equals(allKeys.length));
+                for (var model in models) {
+                  expect(model, isNull);
+                }
+              }),
+            ];
+            return Future.forEach(tests, (f) => f());
+          });
         });
       });
-    });
-  }, onPlatform: {
-    'mac-os': new Skip('Missing ALPN support on MacOS.'),
```

Comment:
Thanks for removing this :+1:

Reply:
Removed this in another spot too, test seems to work